scene/3d/skeleton_3d.cpp

```diff
@@ -865,7 +865,7 @@ void Skeleton3D::set_bone_pose_rotation(int p_bone, const Quaternion &p_rotation
 	const int bone_size = bones.size();
 	ERR_FAIL_INDEX(p_bone, bone_size);
 
-	bones[p_bone].pose_rotation = p_rotation;
+	bones[p_bone].pose_rotation = p_rotation.normalized();
```

Comment:
The issue is on the editor's side, so obviously it must not be normalized here.

Reply:
after looking around the issue is not from the editor's side the main issue stems from the fact the rotation quaternion is not normalized specifically in the skeleton3D node unlike the rest of the engine's 3D Nodes (probably because each individual bone is not regarded as a 3D node and needed to have it's logic made from scratch), and even if the solution was made on the editor side it would only "fix it" in appearance, under the hood the quaternion would still be non normalized.
`.\godot\editor\plugins\skeleton_3d_editor_plugin.cpp`
![Screenshot 2024-10-18 214808](https://github.com/user-attachments/assets/20ae1baf-c07e-4684-aaf3-0c2e0b5abd5a)
*this is what it may look like if the solution was on the editor side*

Reply:
Ah, yes, the lack of normalization in set_bone_pose_rotation is a problem. I thought it already had that, but it seems not.

So this normalization may not be a problem, but it should be tested for consistency with the global transformation by gizmo.